src/commands/utils/modules.ts

```diff
@@ -45,6 +51,23 @@ class JahiaUtilsModule extends Command {
     const dependencies: string[] = flags.dependencies.split(',')
 
     const jahiaFullUrl = flags.jahiaUrl.slice(-1) === '/' ? flags.jahiaUrl : flags.jahiaUrl + '/'
+    for (let i = 0; i < flags.timeout; i++) {
```

Comment:
Why not use a while here ? I feel this would improve readability (and not need a break).